libs/blog-bff/newsletter/src/lib/api.ts

```diff
@@ -0,0 +1,20 @@
+import { Hono } from 'hono';
+import { csrf } from 'hono/csrf';
+
+import { wpNewsletterClientMw } from '@angular-love/util-wp';
+
+const app = new Hono();
+
+app.post('/', wpNewsletterClientMw, async (c) => {
+  const origin = c.req.header('Origin');
+  app.use(
+    csrf({
+      origin: origin,
+    }),
+  );
```

Comment:
I don't think this is right approach. The `origin` typically refers to the allowed origins that can access your bff. By getting the origin from the reuqest, you allow ALL origins to access the bff. 

And I think those `app.use()` statesments should never be inside of a request handler. Look at the examples -> https://hono.dev/middleware/builtin/csrf

Reply:
Resolved earlier. I've had some doubts about it, but right now it's solved